### ISSUE TITLE
New rules raise false warning in `global.disable`

### ISSUE THREAD

**Issue by martimlobao** [rvben/rumdl#261]
The new rules raise a false positive warning saying they are not recognized in `global.disable`, even though adding them to the list correctly disables them.

```toml
# .rumdl.toml
[global]
disable = [
  "MD070",
  "MD071",
  "MD072",
]
```

```bash
❯ rumdl --version
rumdl 0.0.210

❯ rumdl check    
[config warning] Unknown rule in global.disable: MD070 (did you mean: MD010?)
[config warning] Unknown rule in global.disable: MD071 (did you mean: MD021?)
[config warning] Unknown rule in global.disable: MD072 (did you mean: MD032?)
```

**Comment by rvben** [comment]
Thanks @martimlobao! Fixed in v0.0.211 - the new rules are now properly recognized in global.disable.
